publisher-sdk-tests/src/androidTest/java/com/criteo/publisher/BidManagerFunctionalTest.java

```diff
@@ -192,6 +193,42 @@ public void prefetch_GivenAdUnits_ShouldCallCdbAndPopulateCache() throws Excepti
     assertShouldCallCdbAndPopulateCacheOnlyOnce(mappedAdUnitsChunks.get(0), slot);
   }
 
+  @Test
+  public void prefetch_GivenRateLimited_ShouldNotCallCdbAndPopulateCache() throws Exception {
+    when(config.getMinTimeBetweenPrefetchesMillis()).thenReturn(1L);
+    CdbResponseSlot slot = givenMockedCdbRespondingSlot();
+
+    CacheAdUnit cacheAdUnit = sampleAdUnit();
+    AdUnit adUnit = givenMockedAdUnitMappingTo(cacheAdUnit);
+
+    BidManager bidManager = createBidManager();
+    bidManager.prefetch(singletonList(adUnit));
+    waitForIdleState();
+
+    bidManager.prefetch(singletonList(adUnit));
```

Comment:
Do you use the fact the `Clock` is mocked to assume that the prefetch is limited ? If yes, what do you think about adding a comment explaining this ?